src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -516,6 +516,42 @@ public async Task ProxyTunnelRequest_PortSpecified_NotStrippedOffInUri()
             Assert.True(connectionAccepted);
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "UAP HTTP stack doesn't support .Proxy property")]
+        [Fact]
+        [OuterLoop("Uses external server")]
+        public async Task ProxyTunnelRequest_UserAgentHeaderAdded()
+        {
+            if (!UseSocketsHttpHandler)
+            {
+                return; // Skip test since the fix is only in SocketsHttpHandler.
+            }
+
+            // Https proxy request will use CONNECT tunnel, even the default 443 port is specified, it will not be stripped off.
```

Comment:
It's not necessary to specify the 443 port since it is the default for the https server  (Configuration.Http.SecureHost).  Also, I don't think the comment is needed either.